manifest.json

```diff
@@ -5572,6 +5572,29 @@
                 }
             }
         },
+        "Psychpsyo.ContactsMinus": {
+            "name": "Contacts-",
+            "description": "Adds the ability to set your world to Contacts-.",
+            "category": "Misc",
```

Comment:
Seems appropriate for the Memes category

Reply:
I've changed the category to Memes.